docs-content/getting-started/7_native-opentelemetry/5_browser.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Browser Instrumentation
+slug: browser-otel-instrumentation
+heading: OpenTelemetry Browser Instrumentation
+metaTitle: Send OpenTelemetry Browser Instrumentation Data to Highlight
+createdAt: 2024-09-17T00:00:00.000Z
+updatedAt: 2024-09-17T00:00:00.000Z
+quickstart: true
+---
+
+If you want to send OpenTelemetry data to Highlight from [OTeL's Browser instrumentation](https://opentelemetry.io/docs/languages/js/getting-started/browser/) you can do this by making a couple tweaks to your configuration.
+
+### 1. Add span resource atributes
+
+In order to associate spans with your project and sessions in Highlight, you'll need to add a few attributes:
+
+* `highlight.project_id` - Allows us to associate traces with your project.
+* `highlight.session_id` - Allows us to associate traces with a specific session in your project.
+* `service.name` (optional) - Specify the service name you want to assign on your traces. This makes it easier to filter to all traces from this instrumentation.
+
+You can set these attributes when creating your trace provider:
+
+```ts
+const provider = new WebTracerProvider({
+  resource: new Resource({
+    // TODO: Explain how to get the project and session IDs.
+    'highlight.project_id': 'your-project-id',
+    'highlight.session_id': 'your-session-id',
```

Comment:
@Vadman97 have any recommendations on how to tell people to access this? I didn't see anything that allowed you to access just the session ID, only the session URL.

Reply:
Ended up adding `sessionSecureID` to the object returned from `H.getSessionDetails()`. @Vadman97 let me know if you see any issues with that!

Reply:
yeah that makes sense to me - the `H.init()` call also returns it but probably a simpler API to use the `getSessionDetails()` call